src/applications/validate-mhv-account/containers/ErrorMessage.jsx

```diff
@@ -11,38 +11,34 @@ import UpgradeAccountFailed from './UpgradeAccountFailed';
 import { ACCOUNT_STATES } from './../constants';
 import { MVI_ERROR_STATES } from './../../../platform/monitoring/RequiresMVI/constants';
 
-export default class ErrorMessage extends React.Component {
-  render() {
-    const { params } = this.props;
+export default function ErrorMessage({ params }) {
+  let errorCode = params.errorCode
+    ? params.errorCode.replace(/-/g, '_')
+    : undefined;
 
-    let errorCode = params.errorCode
-      ? params.errorCode.replace(/-/g, '_')
-      : undefined;
-
-    // Capitalize in case of mvi error
-    if (errorCode && errorCode.startsWith('mvi_error')) {
-      errorCode = errorCode.replace('mvi_error_', '').toUpperCase();
-    }
+  // Capitalize in case of mvi error
+  if (errorCode?.startsWith('mvi_error')) {
```

Comment:
I converted the `if` to use optional chaining.

Reply:
Good call